content/en/threat-model.md

```diff
@@ -0,0 +1,123 @@
+|                        |                  |               |                       |              | x             |              |             | None: alerts are logged, maybe sent to Slack. Sigstore admins manually check reports from Charlie.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|                        |                  |               | x                     |              | x             |              |             | Same as Fulcio monitor + Rekor monitor                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|                        |                  | x             |                       |              |               |              |             | Can remove certificates (invalidate them) by indicating that they are not in the log when Bob queries, fork attack by showing different times or certificates to different users (for things Fulcio has signed), and may block the issuance of Fulcio certificates by not creating an SCT.  Deletion / fork may be detected by Fulcio CT log monitors.                                                                                                                                                                                                   |
+|                        |                  |               |                       | x            |               |              |             | Can change the timestamp on Fulcio signatures used outside the validity window to make them appear valid when Bob queries.  Can deny the existence of valid signatures leading to them being untrusted.  Can later recover using offline delegation in Sigstore’s TUF root of trust.  Deletion or fork attacks may be detected by Rekor monitors. Changes to the Rekor CT log would be challenging to recover as it eliminates the append-only promise of Rekor.                                                                                         |
```

Comment:
This section is a bit hard to follow.  In particular:

* Some sentences are written from an attacker PoV, and some are written from a defender PoV ("Can later recover ..."), but it's not immediately clear which are which.
* There's a section on "Deletion or fork attacks", but those attacks aren't mentioned in this section.
* In this scenario, the attacker has access to the Rekor server, not the Recor CT log, but it ends with "Changes to the Rekor CT log ...". Does access to the Rekor server imply write / rewrite access to the CT log?  (If so, state that explicitly.)

Reply:
(There's no separate item for Rekor CT log, unlike Fulcio)